From e2594bc6b64e48eebae5de31612f764a46d521b4 Mon Sep 17 00:00:00 2001
From: Kevin Mark <kmark937@gmail.com>
Date: Tue, 2 Jul 2019 14:21:22 -0400
Subject: [PATCH 08/49] do not bother running ldconfig on DESTDIR.  it wants to
 write the temp cache file outside of the chroot.  doesnt matter anyways as we
 wont use the cache results (portage will rebuild cache), so running ldconfig
 is simply a waste of time.

---
 Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Makefile b/Makefile
index 61cfa436c4..ab8e65c861 100644
--- a/Makefile
+++ b/Makefile
@@ -116,6 +116,7 @@ install-symbolic-link: subdir_install
 	rm -f $(symbolic-link-list)
 
 install:
+dont-bother-with-destdir:
 	-test ! -x $(common-objpfx)elf/ldconfig || LC_ALL=C LANGUAGE=C \
 	  $(common-objpfx)elf/ldconfig $(addprefix -r ,$(install_root)) \
 				       $(slibdir) $(libdir)
-- 
2.22.0

